index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Lato" />
+  <link rel="stylesheet" href="./css/mob-style.css">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Tomorrowland</title>
+</head>
+<body>
+  <header>
+    <nav class="nav">
+        <div class="mob-nav">
+          <img src="images/menu-icon2.png" id="open-menu" onclick="display()" alt="menu option">
+          <img src="images/close.png" id="close-menu" onclick="display()" alt="Close Icon">
+        </div>
+        <div class="mob-nav-container" id="menu">
+          <ul class="mob-nav-ul">
+            <li>
+              <a href="index.html" onclick="display()">Home</a>
+            </li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+          </ul>
+        </div>
+        <div class="top-nav">
+          <div class="logo">
+            <a href="index.html"><img src="/images/tomorrowland.png" alt="Logo"></a>
+          </div>
+          <ul class="main-nav">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Desktop Size:
- Pease the Navbar is divided in a top dark bar and a bottom white bar.
- The top short bar section contains social media links, logout, and language links aligned to the right.
- The bottom bar contains a logo and a list of page links.
- See Original Design:
![Capture](https://user-images.githubusercontent.com/63972670/180992530-775e58af-a633-48bd-94aa-833d3479e15b.PNG)
- See your Design: 
![1](https://user-images.githubusercontent.com/63972670/180992654-4eb99f6e-725e-4837-ada8-1d5bf5ed69f3.PNG)

- [ ] Kindly fix the logo image and make sure the logo is located on the left side and the pages links are aligned to the right